sklearn/linear_model/_stochastic_gradient.py

```diff
@@ -1855,9 +1859,11 @@ class SGDRegressor(BaseSGDRegressor):
     eta0 : float, default=0.01
         The initial learning rate for the 'constant', 'invscaling' or
         'adaptive' schedules. The default value is 0.01.
+        Values must be in the range `(0.0, inf)`.
```

Comment:
This seems incorrect if I look at the parameter validation.

```suggestion
        Values must be in the range `[0.0, inf)`.
```

Reply:
We probably need to correct also the classifier docstring.

Reply:
> We probably need to correct also the classifier docstring.

@glemaitre You're right. I fixed this as well. Thank you for catching all of these.